adafruit_requests.py

```diff
@@ -366,6 +360,60 @@ def __init__(
         self._session_id = session_id
         self._last_response = None
 
+    def _build_boundary_data(self, files: dict):
+        boundary_string = self._build_boundary_string()
+        content_length = 0
+        boundary_objects = []
+
+        for field_name, field_values in files.items():
+            file_name = field_values[0]
+            file_data = field_values[1]
```

Comment:
the name `file_data` makes me think that it's going to contain the actually data from the file rather than the FileIO stream object thing (not certain of the proper name) that allows you to `read()` the data, but isn't the actual data itself. 

In the original code I think I was using `file_handle` for this concept which fits closer in my mind to what the variable is, but there may be something better than that as well. 

In any case, I'm in favor of something other than `_data` suffix for this

Reply:
Hmmm, it's either a `string` or a `handle`... I hate naming. If everything else is good, I'll thing through this as I add tests.

Reply:
Ah, that's true. I was thinking only about the file case, not the "simple value" case with just a string.